Terminal.Gui/Views/Shortcut.cs

```diff
@@ -631,7 +631,7 @@ private void UpdateKeyBinding ()
     ///     - if the user presses the HotKey specified by CommandView
     ///     - if HasFocus and the user presses Space or Enter (or any other key bound to Command.Accept).
     /// </summary>
-    protected new bool? OnAccept (CommandContext ctx)
+    protected bool? OnAccept (CommandContext ctx)
```

Comment:
Is the nullable necessary here?

Reply:
I think so. 

Part of the design of Command/KeyBinding is described here.  For this, I believe we need to support propogating "don't know/yes/no" up and down the stack. OnAccept is right in the middle of all this.

https://github.com/gui-cs/Terminal.Gui/blob/fd683d83f5c279129c7f5ba5345c6c9ba92e2d1e/Terminal.Gui/View/ViewKeyboard.cs#L653-L658

Reply:
Works for me.

An opportunity for a small future optimization is there, but it's fairly minor.

I also have been starting to think more about accessibility modifiers, as we get closer to completion. This one, being a kinda non-standard handler, because of not being void, might be a candidate for `private protected` instead of just `protected`.

The resulting visibility is the intersection of protected and internal, so it's visible to derived types in the same assembly only.

Reply:
That also lets it continue to do what it needs to do, without having to worry about it being overridden outside of the library.